ports/glutin/window.rs

```diff
@@ -645,13 +631,24 @@ impl WindowMethods for Window {
     /// Helper function to handle keyboard events.
     fn handle_key(&self, key: Key, mods: constellation_msg::KeyModifiers) {
 
+        // This is prefixed with _ because the compiler will consider it unused otherwise.
+        // It isn't a constant because the compiler does not allow user-defined operations
+        // in constants.
+        let _control_and_alt = CMD_OR_CONTROL | ALT;
         match (mods, key) {
-            (_, Key::Equal) if mods & !SHIFT == CMD_OR_CONTROL => {
-                self.event_queue.borrow_mut().push(WindowEvent::Zoom(1.1));
+            (_, Key::Equal) => {
+                if mods & !SHIFT == CMD_OR_CONTROL {
+                    self.event_queue.borrow_mut().push(WindowEvent::Zoom(1.1));
+                } else if mods & !SHIFT == _control_and_alt {
+                    self.event_queue.borrow_mut().push(WindowEvent::PinchZoom(1.1));
+                }
             }
             (CMD_OR_CONTROL, Key::Minus) => {
                 self.event_queue.borrow_mut().push(WindowEvent::Zoom(1.0 / 1.1));
             }
+            (_control_and_alt, Key::Minus) => {
```

Comment:
This is incorrect; it's equivalent to a wildcard.  It binds to a new variable named `_control_and_alt` rather than comparing to the existing one.